docs/DeveloperGuide.md

```diff
@@ -634,6 +634,155 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: UC17 - Add a lesson**
+
+**MSS**
+
+1. User requests to add a lesson.
+2. User provides the parameters of the lesson.
+3. Tutor's Pet adds the lesson.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The parameters provided are invalid.
+
+    * 2a1. Tutor's Pet shows an error message.
+
+      Use case resumes at step 1.
+
+* 3a. The lesson already exists.
+
+    * 3a1. Tutor's Pet shows an error message.
+
+      Use case resumes at step 1.
```

Comment:
Just noticed this, but perhaps this should be `use case ends` since it has been deemed that the lesson already exists?
```suggestion
      Use case ends.
```

Seems like this was inherited from previous versions.

Reply:
Could be a typo on the user's part. Would recommend keeping the ```Use case resumes at step 1.```